Ground overlays s3

gitriley opened the pull request:
Note: the refactoring we did to remove the "displayOverlay" attribute seems to have created a problem when dragging polygons and polylines. Now, when you drag a polygon/polyline, the vertices move, but the overlay itself remains in the same place until you finish dragging. This wasn't happening before.

vanwars commented:
OK! Keep me posted.

On Feb 8, 2018 3:48 PM, "Riley Flynn" <notifications@github.com> wrote:

> actually never mind. The problem seems to have began when I added the
> dragend-save functionality. I'll see if i can pinpoint what's causing it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/LocalGround/localground/pull/170#issuecomment-364287929>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABKGw4Q4Cz687-NXrYJ6w9RrwqUicIK4ks5tS4fHgaJpZM4R_K_k>
> .
>

gitriley commented:
Fixed several bugs and now there is no latency when dragging shapes, and no duplication of event listeners (listeners are being properly cleared). Also, while I was there, I added the ability to delete vertices from a shape via a right click. We didn't discuss this, but it seems fairly necessary. 

For deleting a vertex, we could also consider other user interactions. Options include: 1) a doubleclick, 2) create a little pop-up bubble above the selected vertex and provide a delete button.

Overall, though, I think the right click works well for now.
